src/drawers/rectDrawer.ts

```diff
@@ -33,7 +33,8 @@ export module _Drawer {
       return this._labelsTooWide;
     }
 
-    public drawText(data: any[], attrToProjector: AttributeToProjector, userMetadata: any, plotMetadata: Plot.PlotMetadata) {
+    public drawText(data: any[], attrToProjector: AttributeToProjector, boundingBox: D3.Selection, userMetadata: any,
+        plotMetadata: Plot.PlotMetadata) {
```

Comment:
?
Note that the renderArea from the drawer is gotten from the renderArea from the plot, which I believe was the reliable source of information for visible data